maintainers/maintainer-list.nix

```diff
@@ -13476,6 +13476,12 @@
     githubId = 38314551;
     name = "Peter Okelmann";
   };
+  pokon548 = {
```

Comment:
Split.

Reply:
Can you describe this a little bit? Did you mean I should split this change to individual pull request? Thanks

Reply:
The maintainers list should be updated in a separate commit (no need for a PR).

Reply:
> The maintainers list should be updated in a separate commit (no need for a PR).

It seems already in separate commit now, thanks! :)